src/main/resources/site/pages/default/default.ts

```diff
@@ -363,8 +363,9 @@ function prepareStructuredData(metaInfo: MetaInfoData, page: DefaultPage): Artic
     headline: metaInfo.metaInfoTitle,
     datePublished: metaInfo.metaInfoSearchPublishFrom,
     dateModified: page.data.showModifiedDate?.dateOption?.modifiedDate
-      ? page.data.showModifiedDate.dateOption.modifiedDate +
-        (page.data.showModifiedDate.dateOption.showModifiedTime ? 'T' + new Date().toISOString().split('T')[1] : '')
+      ? `${page.data.showModifiedDate.dateOption.modifiedDate}${
```

Comment:
Tror ikke du trenger egen logikk for om showModifiedTime er valgt, det er kun tenkt for om det skal vises på siden. Så tror du kan prøve denne koden:
`dateModified: page.data.showModifiedDate?.dateOption?.modifiedDate
      ? new Date(page.data.showModifiedDate.dateOption.modifiedDate).toISOString()
      : undefined,`